hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/io/TestHoodieTimelineArchiver.java

```diff
@@ -243,6 +243,7 @@ private HoodieWriteConfig initTestTableAndGetWriteConfig(boolean enableMetadata,
         .withFileSystemViewConfig(FileSystemViewStorageConfig.newBuilder()
             .withRemoteServerPort(timelineServicePort).build())
         .withMetadataConfig(HoodieMetadataConfig.newBuilder().enable(enableMetadata)
+            .withMetadataIndexPartitionStats(false)
```

Comment:
Could we have a config to turn off MDT as a whole instead of turning off each MDT index individually?